tools/depends/target/gnutls/Makefile

```diff
@@ -28,6 +24,10 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 	cd $(PLATFORM); patch -p1 -i ../size-max.patch
 	cd $(PLATFORM); patch -p1 -i ../add-dl-as-private-lib.patch
+ifeq ($(OS),darwin_embedded)
+	# Apple CCAS requires stripping ; from comment block # to build successfully
```

Comment:
This is should be upstreamed right?

Reply:
I honestly dont think it would be accepted. It only affects ios/tvos build tools (not osx), and the fault is incorrectly processing a comment block. For whatever reason, its still trying to process control elements inside the comment.

In addition its a license header comment block, thats in every other source file in the lib. I cant see a gnu project changing a license header for the sake of a closed, niche platform.

Reply:
Fair point.

Reply:
Turns out the issue stemmed from gas-preprocessor. Added commit that fixes that.
Not sure how that effects your ffmpegdirect stuff @phunkyfish